docs/topics/mta-web-administrator-view.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="mta-web-administration-view_{context}"]
+= User Interface views
+
+The {ProductName} ({ProductShortName}) {WebName} and has two user interfaces with different functionalities:
+
+* Administrator view
+* Developer view
+
+== Administrator view
+The administrator view is used by administrators to create the {ProductShortName} {WebName} instance environment.
```

Comment:
Not really to create an instance, but for managing the configuration of an instance.

Reply:
I made the requested changes, please review.